taipy/gui/viselements.json

```diff
@@ -1679,7 +1679,7 @@
                         "default_property": true,
                         "type": "dynamic(str)",
                         "default_value": "\"\"",
-                        "doc": "The message displayed in the notification. Can be a dynamic string."
+                        "doc": "The message displayed in the notification."
```

Comment:
Change the type also

Reply:
Well... that's correct in my opinion... Am I wrong?
Code: `const message = useDynamicProperty(props.message, props.defaultMessage, "");`

Reply:
Sorry, without the context (on my phone) I thought it was about the notification message